recipe/meta.yaml

```diff
@@ -147,10 +146,15 @@ outputs:
         - {{ pin_subpackage("pocl-remote", exact=True) }}
         - {{ pin_subpackage("pocl-cuda", exact=True) }}         # [enable_cuda]
     test:
+      requires:
```

Comment:
Use `test_downstreams: - pyopencl` instead of requiring `pyopencl` and making a cycle in dependency chain.